Models/Connect/ProductToRemoteCategoryRepository.php

```diff
@@ -51,12 +51,17 @@ public function findArticlesByRemoteCategory($remoteCategoryKey = null, $shopId,
             'md.number as Detail_number',
             's.name as Supplier_name',
             'pci.purchasePrice as Price_basePrice',
+            '(p.price * (1 + (t.tax / 100)) as Price_price',
```

Comment:
Do we need to calculate vat manually? I think p.price already contains tax. Have you checked that ?